css/package-lock.json

```diff
@@ -10,20 +10,20 @@
       "integrity": "sha512-3OOR92FTc2p5/EcOzPcXp+Cbo+3C15nV9RXHlOUBCBpHhcB+0frbSNR9ehED/o7sTcyGVtqGJpguToEdlXhD0w==",
       "dev": true,
       "requires": {
-        "convert-source-map": "^1.5.0",
-        "glob": "^7.0.5",
-        "indx": "^0.2.3",
-        "lodash.clone": "^4.3.2",
-        "lodash.defaults": "^4.0.1",
-        "lodash.flatten": "^4.2.0",
-        "lodash.merge": "^4.4.0",
-        "lodash.partialright": "^4.1.4",
-        "lodash.pick": "^4.2.1",
-        "lodash.uniq": "^4.3.0",
-        "resolve": "^1.5.0",
-        "semver": "^5.3.0",
-        "uglify-js": "^2.8.22",
-        "when": "^3.7.8"
+        "convert-source-map": "1.6.0",
```

Comment:
Tieto zmeny nepatria do tohto PR. Mozes ich prosim poslat ako zvlast PR? Kvoli lahsej revertovatelnosti.

Reply:
Prepac, ale vobec neviem, co to je. Tipol, by som, ze je to vysledok kompilacie toho .less suboru alebo to vzniklo pri mergovani noveho mastra u mna. Mam na to teda vyrobit novy branch a v nom PR? Co mam dat do commit message (kedze neviem, co to robi).

Reply:
Vyzerqa ze to sposobilo `npm install` Urobil som PR tu: https://github.com/trojsten/web/pull/1312